packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -0,0 +1,121 @@
+import bytes from 'bytes';
+// @ts-ignore
+import { createDeployment, createLegacyDeployment } from 'now-client';
+import wait from '../output/wait';
+import createOutput from '../output';
+
+export default async function processDeployment({
+  now,
+  debug,
+  hashes,
+  paths,
+  requestBody,
+  uploadStamp,
+  legacy,
+  env,
+  quiet
+}: any) {
```

Comment:
Can you create a new interface for this object instead of `any`?

Reply:
I tried, but `Now` type wasn't resolving properly 🤔
I'll take a look